### DISCUSSION TITLE
Rough Estimate as to When the ADC Fixes Will be Released?

### DISCUSSION THREAD

**Discussion by btribley** [SpenceKonde/DxCore#73] [Future development discussion]
OK, I will admit that I am being a wimp. Atmel Studio 7 will import a sketch but only if it was installed with board manager. Visual Micro makes it easy to create an Arduino project, again if the hardware is in board manager. I'm about 12 hours into trying to get an unofficial release of DxCore to show up and build properly in these tools. If I had a board manager version I think I would be on my way.

This is a hobby project. I would love to just end this odyssey (although I have learned a lot) use another chip. Problem is I need a small, low-power 8-bit MCU with a 12 bit ADC and I don't want to fudge it with averaging, etc. I have migrated from an 8-bit PIC microcontroller that has a 12 bit ADC, my application demands the precision. The other thing I need is more program space. The Dx chips are perfect. 

So it would help me at this point to have a rough idea as to the timing of the next release. Is it weeks or months? Thanks in advance for any information you can give.

**Comment by SpenceKonde** [comment]
1.3.1 should be released today.

**Reply by btribley** [reply]
:+1:

**Comment by SpenceKonde** [comment]
Oh, and this was released sunday. Not sure whether I said this through some other channel aimed at you.

**Reply by btribley** [reply]
I saw it on Monday morning and had to will myself to ignore it while doing my day job... I should have encouraged you to take the day off on Sunday instead of egging you on to finish, my apologies. Great work! I am still wrapped around the tools axle a bit but I tested the release against one of my simple scripts and the ADC is working great. Thank you for all your effort on this!
